### ISSUE TITLE
Is the search broken or I am doing sth wrong

### ISSUE THREAD

**Issue by TheRealTechWiz** [Blatzar/shiro-app#36]
**Describe the bug**
I searched pokemon (2019) it showed no results. But I found it in the recently uploaded section. Not sure if the issue is specifically for pokemon. But that is what is tried it for.

**To Reproduce**
Search "Pokemon (2019)" in search section.
It will show no results.
Go to recently uploaded section scroll down and find it.

**Screenshots**
![Search Section](https://user-images.githubusercontent.com/42109004/120914523-5d83c480-c6bc-11eb-9355-d7d24a4df563.jpeg)

![Recently Uploaded](https://user-images.githubusercontent.com/42109004/120914521-5bba0100-c6bc-11eb-9ecd-f63bf84b0324.jpeg)

**Please complete the following information:**
 - Android version: Android 10
 - App version: 2.2.3 (android phone not TV)
 - Phone model : Poco M2

**Additional context**

**Comment by Blatzar** [comment]
Thank you for reporting with pictures and everything, it really helps! You are not doing anything wrong, the site does not give Pokemon 2019 either. I'll talk to the site dev to try to get this fixed :)

**Comment by TheRealTechWiz** [comment]
It's great to see that you are so active and taking criticism constructively to improve this app.


> the site does not give Pokemon 2019 either

If you are taking about shiro.is site then I searched there and it did return the anime in return.
![image](https://user-images.githubusercontent.com/42109004/120935159-0e22b000-c71f-11eb-9685-768329505829.png)

**Comment by Blatzar** [comment]
Only the dubbed version however, the reason that's not showing is because the app hides all dubbed shows by default (can be disabled in settings).
The site should also show the subbed one :pensive:

**Comment by TheRealTechWiz** [comment]
Yes you are correct. I disabled hide all dubbed shows in settings now it is showing.

> 😔

That's the problem with the site not app. Your app is amazing. This is one of the best app for android TV.
